README.md

```diff
@@ -125,3 +125,7 @@ chromium-browser --disable-web-security --user-data-dir=/home/user/anywhere/chro
 This works because there is code in app.tsx that preempts all fetch() calls and checks if the app is being served from localhost. If so, it modifies the request to point to localhost:8080 specifically, rather than localhost:9000 which is where webpack-dev-server serves the frontend code from.
 
 It might be possible to improve this technique. Suggestions are welcome.
+
+### End user documentation
+
+The source for the end user documentation is stored in /pantheon-bundle/src/main/resources/SLING-INF/content/docs/ref_prerequisites.adoc.
```

Comment:
I'd just mark up the path in monospace, other than that, this is great. Thank you! :+1: